### PR TITLE
Fixed issue with dash in the room name causing Jitsi Meet to throw a 404 error

### PR DESCRIPTION
**Pull request by ChristineBoersen**
Fixed issue with dash in the room name causing Jitsi Meet to throw a 404 error

### CODE REVIEW

**Comment by ChristineBoersen** [comment]
This should resolve Issue #1

File: slashcommand.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class JitsiSlashCommand implements ISlashCommand {
 
         const [roomName] = context.getArguments();
 
-        const url = server + (roomName ? roomName : context.getRoom().id + '-' + context.getSender().id);
+        const url = server + (roomName ? roomName : context.getRoom().id + context.getSender().id);
```
Comment (graywolf336) [inline slashcommand.ts]: Please remove the extra whitespace at the end. :)

Comment (ChristineBoersen) [reply slashcommand.ts]: Ok I think I have it cleaned up now (Forgot that I'm used to a very different version control system) :)

**Comment by saschafoerster** [comment]
At least this little bug is documented! Found it myself yesterday and it is a bit annoying.

**Comment by ChristineBoersen** [comment]
What's more annoying is I did the pull request that fixes it over 2 months ago, for one line of code, and it still isn't implemented.

**Pull request merged**